great_expectations/util.py

```diff
@@ -889,6 +891,14 @@ def is_float(value: Any) -> bool:
     return True
 
 
+def is_parseable_date(value: Any, fuzzy: bool=False) -> bool:
+    try:
+        parsed_date: datetime = parse(value, fuzzy=fuzzy)
```

Comment:
I'm not sure if how well strings that `datetime.parse` can parse will align with the various engines that the datetime string will be fit into. But this seems to work for pandas.